can_tools/scrapers/official/MD/md_state.py

```diff
@@ -23,7 +23,7 @@ def normalize(self, data):
         df = self.arcgis_jsons_to_df(data)
         df.columns = [x.lower() for x in list(df)]
         df["location"] = self.state_fips
-        df = df[df["filter"] == "s"].reset_index(drop=True)
+        df = df.tail(1).reset_index(drop=True)
```

Comment:
@ChrisBremer this looks a bit "risky". How can we be sure that we are getting the row we want? Do we have guarantees of sortedness?

Reply:
I've only ever seen it sorted but you're right - updated to presort